arcscript.js

```diff
@@ -53,10 +53,6 @@ $(document).ready(function(){
 	var hzIInames = $(".character.horII");
 	var miscnames = $(".character.misc");
 
-	var intersections = $(".blue2, .orange2, .green2, .red2, .gold2, .purple2, .silver2, .black2");
-	var singleselect = $(".blue1, .orange1, .green1, .red1, .gold1, .purple1, .silver1, .black1");
-	var deselect = $(".blue, .orange, .green, .red, .gold, .purple, .silver, .black");
```

Comment:
these variables are only used in click handlers, and they need to get re-queried there anyway, so they can just define their own variables. Otherwise, if they all share the same set of variables, it's possible for a click handler to accidentally use the "older" set of intersections, single selections, etc.